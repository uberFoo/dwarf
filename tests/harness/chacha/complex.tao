struct ComplexEx;

fn main() {
    ComplexEx::this_will_fail();
}
