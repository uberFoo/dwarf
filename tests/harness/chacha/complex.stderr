Interpreter exited with:
[31mError:[0m no such static method
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0m complex:4:5 [38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m4 â”‚[0m [38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[31mC[0m[31mo[0m[31mm[0m[31mp[0m[31ml[0m[31me[0m[31mx[0m[31mE[0m[31mx[0m[31m:[0m[31m:[0m[31mt[0m[31mh[0m[31mi[0m[31ms[0m[31m_[0m[31mw[0m[31mi[0m[31ml[0m[31ml[0m[31m_[0m[31mf[0m[31ma[0m[31mi[0m[31ml[0m[31m([0m[31m)[0m[38;5;249m;[0m
 [38;5;240m  â”‚[0m     [31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”¬[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m  
 [38;5;240m  â”‚[0m                  [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m in this invocation
 [38;5;240m  â”‚[0m 
 [38;5;240m  â”‚[0m [38;5;115mNote[0m: [33mComplexEx[0m does not have a static method named [33mthis_will_fail[0m
[38;5;246mâ”€â”€â”€â•¯[0m

